features/window-management.yml

```
@@ -0,0 +1,24 @@
+name: Window Management
+description: "The Window Management API makes it possible to build multi-screen applications by providing access to information about the screens on a device, and to place content on a specific screen."
```

More concretely, I think we could say something like:

```suggestion
description: "The `window.getScreenDetails()` method gets information about the screens on a device, which you can use to place new windows on a specific screen with `window.open()`.
```